api/src/main/java/org/openmrs/module/fhir2/api/translators/CodingTranslator.java

```diff
@@ -33,5 +41,26 @@ public interface CodingTranslator extends ToFhirTranslator<Concept, Coding>, ToO
 	 * @return the corresponding OpenMRS {@link Concept}
 	 */
 	@Override
-	Concept toOpenmrsType(@Nonnull ICoding coding);
+	public Concept toOpenmrsType(@Nonnull ICoding coding) {
```

Comment:
changed CodingTranslator from an interface -> abstract class to avoid duplicating the toOpenmrsType method. It feels like this toOpenmrsType() implementation is generic enough that it can be used by most CodingTranslator extending classes.

`getCodingForSystem` is another method that i think could be usefule for any CodingTranslator

Reply:
happy to revert back to the previous interface definition if need be, though